pull/github.go

```diff
@@ -346,6 +347,25 @@ func (ghc *GitHubContext) LatestStatuses() (map[string]string, error) {
 	return ghc.statuses, nil
 }
 
+func (ghc *GitHubContext) Labels() ([]string, error) {
+	if ghc.labels == nil {
+		var labels []string
+		issueLabels, _, err := ghc.client.Issues.ListLabelsByIssue(ghc.ctx, ghc.owner, ghc.repo, ghc.number, &github.ListOptions{
+			Page:    0,
+			PerPage: 1,
```

Comment:
Will this fail if the PR contains more than 1 label? I can understand not supporting arbitrary paging for labels, but it feels like this should be higher, maybe using the maximum page size of 100?